ui/segment.reel/segment.css

```diff
@@ -0,0 +1,12 @@
+.montage-Segment {
+    /* to vertical align text labels */
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+
+    -webkit-box-align: center;
+    -webkit-align-items: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
```

Comment:
I think -webkit-box is needed for Android browser 4.3... are we supporting Android browsers before 4.4?